include/pycppad/codegen/cg.hpp

```diff
@@ -7,105 +7,125 @@
 
 #include <cppad/cg/cppadcg.hpp>
 
-#include "eigenpy/user-type.hpp"
-#include "eigenpy/ufunc.hpp"
+#include <eigenpy/user-type.hpp>
+#include <eigenpy/ufunc.hpp>
 
+#include "pycppad/cast.hpp"
 
 namespace pycppad
 {
-  namespace codegen {
+  namespace internal
```

Comment:
We should move the Cast struct to pycppad/codegen/cast.hpp to be consistent with pycppad/cast.hpp

Reply:
I can handle that, even if I don't think it will lead to a sensitive impact. It will mostly add multiple inclusion and so longer compilation times.
I would keep it here and change later if needed.
Do you agree?

Reply:
Sure no problem